Mathlib/Algebra/CharP/Basic.lean

```diff
@@ -442,6 +442,27 @@ theorem sum_pow_char {ι : Type*} (s : Finset ι) (f : ι → R) :
   (frobenius R p).map_sum _ _
 #align sum_pow_char sum_pow_char
 
+variable (n : ℕ)
+
+theorem list_sum_pow_char_pow (l : List R) : l.sum ^ p ^ n = (l.map (· ^ p ^ n : R → R)).sum := by
+  induction' n with n ih
```

Comment:
Can you use `induction` rather than `induction'`? Sooner or later `induction'` will be deprecated.